server/events/vcs/testdata/github-commit-status-full.json

```diff
@@ -1,6 +1,30 @@
 {
     "state": "blocked",
     "statuses": [
+      {
```

Comment:
@andrewjamesbrown are you planning to contribute a fix for this issue or is this only a failing test contribution? If it's only a contribution of a failing test then I don't think it will be added to the code base.